components/eam/src/dynamics/se/dyn_comp.F90

```diff
@@ -317,7 +324,21 @@ subroutine dyn_init2(dyn_in)
           ! new run, scale mass to value given in namelist, if needed
           call prim_set_mass(elem, TimeLevel,hybrid,hvcoord,nets,nete)
        endif
+       call cldera_dynamic_tracers_pv_idx(pv_idx)
```

Comment:
It looks like this code exists to add the offset to make sure PV is non-negative, is that correct? If so, can you please add a comment here explaining this?

Reply:
@OkayHughes can you please comment?

Reply:
This particular line uses a helper routine which will return 0 if pv_idx should not be used, otherwise it returns the index for the PV tracer in the dynamics Q variable. The next loop initializes PV (otherwise Q(:,:,:,pv_idx) will be identically zero). An offset is added (which is a parameter read from the tracer module) to prevent negative tracer values, which is subtracted at output time. This parameter is chosen to minimize floating point cancellation. While vertical variation of this constant would cause less cancellation where PV is largest, this would change the advected values. The current value is a compromise.

Reply:
I added a comment which describes this